handbook/quality/README.md

```diff
@@ -0,0 +1,145 @@
+
+### Reproduced
+QA has reproduced the issue successfully. It should now be transferred to engineering to work on. 
+
+Remove the “reproduce” label and add the label of the relevant team (#agent, #platform, #interface) and assign it to the relevant engineering manager (make your best guess as to which team – the EM will re-assign if they think it belongs to another team). Move it to “Ready” in the Release board. [See on Github](https://github.com/fleetdm/fleet/issues?q=archived%3Afalse+org%3Afleetdm+is%3Aissue+is%3Aopen+label%3Abug+-label%3A%3Areproduce+-project%3Afleetdm%2F37+project%3Afleetdm%2F40+-assignee%3Axpkoala+sort%3Aupdated-asc).
+
+### Orphans 
+Bugs which do not have the reproduce label and do not exist on the release board. This filter serves as a sanity check. There should be no bugs in this state, because it means this bug is likely to be forgotten by our process. [See on Github](https://github.com/fleetdm/fleet/issues?q=archived%3Afalse+org%3Afleetdm+is%3Aissue+is%3Aopen+sort%3Aupdated-asc+label%3Abug+-label%3A%3Areproduce+-project%3Afleetdm%2F37+-project%3Afleetdm%2F40+).
+
+### Reproduced orphans 
+Bugs which do not have the reproduce label and do exist on the release board, but do not have one of the three teams tagged. There should be no bugs in this state. This will risk being forgotten by the process because it does not appear in any of the standard team-based filters, which means it risks never being seen by engineering. [See on Github](https://github.com/fleetdm/fleet/issues?q=archived%3Afalse+org%3Afleetdm+is%3Aissue+is%3Aopen+sort%3Aupdated-asc+label%3Abug+-label%3A%3Areproduce+-project%3Afleetdm%2F37+project%3Afleetdm%2F40+-assignee%3Axpkoala+-label%3A%23interface+-label%3A%23platform+-label%3A%23agent+).
+
+### All bugs
+[See on Github](https://github.com/fleetdm/fleet/issues?q=is%3Aissue+is%3Aopen+label%3Abug).
+
+## Rituals
+
+Directly Responsible Individuals (DRI) engage in the ritual(s) below at the frequency specified.
+
+| Ritual                       | Frequency                | Description                                         | DRI               |
+|:-----------------------------|:-------------------------|:----------------------------------------------------|-------------------|
+| Bug Review                   | Weekly                   | Review bugs that are in QA's inbox                  | Reed Haynes       |
+
+## Slack channels
+
+This group maintains the following [Slack channels](https://fleetdm.com/handbook/company#why-group-slack-channels):
+
+| Slack channel                       | [DRI](https://fleetdm.com/handbook/company#why-group-slack-channels)|
+|:------------------------------------|:--------------------------------------------------------------------|
+| `#help-qa`                          | Reed Haynes                                                         |
+| `#help-release`                     | Reed Haynes                                                         |
```

Comment:
Shall we consider condensing this into #help-engineering?

Reply:
I think one place to see and discuss and organize release bugs would be useful (#help-release). I don't feel strongly about #help-qa. That's left over from before.